src/Components/Dialogs/ConfirmImportDialog.js

```diff
@@ -48,13 +63,15 @@ export default function ConfirmationDialog(props) {
     try {
       await patchPublication(publication);
       await patchPiaPublication(publication.cristinResultId, publication.pub_id);
-    } catch (e) {
-      console.log('There was an error while updating the publication', e);
-      if (!e.hasOwnProperty('response') || e.response.status === 401 || e.response.status === 403) {
+      setImportDisabled(false);
+    } catch (error) {
+      console.log('There was an error while updating the publication', error);
+      if (!error.hasOwnProperty('response') || error.response.status === 401 || error.response.status === 403) {
         localStorage.setItem('authorized', 'false');
         history.push('/login');
```

Comment:
Egen generell api funksjon for å sjekke om sessionen har gått ut

Reply:
ble gjort i del 1